tests/test_crs/valid_fiod.yaml

```diff
@@ -7,8 +7,9 @@ spec:
   elasticsearch:
     server: "marquez.perf.lab.eng.rdu2.redhat.com"
     port: 9200
+    index_name: ripsaw-fio
   metadata:
-    collection: true
```

Comment:
collection should be set to true still

Reply:
ouch!, I forgot to revert this change after running test locally :)